pkgs/by-name/ru/rustic/package.nix

```
@@ -17,25 +17,23 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-HYPzgynCeWDRRNyACHqnzkjn6uZWS0TDHuJE9STJxbQ=";
   };
 
-  useFetchCargoVendor = true;
   cargoHash = "sha256-+BlLVnvI2qBfwEtyxmZFNhR9MEzs0/a1Ce6ALOKtoPU=";
 
   nativeBuildInputs = [ installShellFiles ];
 
   postInstall = ''
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlaform) ''
```
https://github.com/NixOS/nixpkgs/issues/308283

---

This is a good catch, and yes, I think it is fine to do that in this PR.